### ISSUE TITLE
Prado Rd. Development

### ISSUE THREAD

**Issue by 888Akasha** [SLUGIS/CAD#179]
New development not in CAD

![image](https://cloud.githubusercontent.com/assets/6013706/16865458/c5c621a2-4a19-11e6-866b-e1463f1b1881.png)

![image](https://cloud.githubusercontent.com/assets/6013706/16865472/df0c32f0-4a19-11e6-97c8-363c981c957d.png)

**Comment by oeon** [comment]
@888Akasha yes, I think we've done a pretty good job of mapping this in OSM but apparently we need to get this data to CAD? George lives around there - so that's helped. Glad to see you referred to OSM! Cc @hpanno

**Comment by 888Akasha** [comment]
This is a big one for me... way fancy...

**Comment by 888Akasha** [comment]
@oeon How do you get the road label on 2 lines? Every time I try it runs it all together on 1 line.

**Comment by oeon** [comment]
@888Akasha `[Field1] & vbnewline & [Field2]` https://geonet.esri.com/thread/48202 ?

**Comment by 888Akasha** [comment]
@oeon I can't get this to work! @hpanno can fix labels on County side!!

**Comment by oeon** [comment]
@hpanno can you put together an expression for @888Akasha? or maybe make a quick gif on this?

not sure what you mean "fix labels on County side"

**Comment by 888Akasha** [comment]
@oeon I haven't done expressions before. I don't know where to put it. I tried Google but couldn't figure it out. @hpanno has to redo this on the county server right? I should probably know how to do it though...

**Comment by oeon** [comment]
@hpanno's got your back. I think you'll do this in the client/admin map .mxd

**Comment by hpanno** [comment]
@oeon sure thing! @888Akasha I'll follow up in a few with a fix!

**Comment by hpanno** [comment]
@888Akasha The following is if you would like the contents of one field to be stacked in the label: 

http://i.giphy.com/7TsSSKS8xnYSA.gif

1. Make sure the Labeling Toolbar is activated.
2. Layer Properties> Placement Properties> Fitting Strategy> "Stack Labels" should be checked.

Did this help?
